includes/class-wc-payments.php

```diff
@@ -327,32 +323,6 @@ private static function is_at_plugin_install_page() {
 		return 'update' === $cur_screen->id && 'plugins' === $cur_screen->parent_base;
 	}
 
-	/**
-	 * Checks whether Woo Multi-Currency is disabled and displays admin error message if enabled.
-	 * TODO: Once Multi-Currency support is implemented, remove this check.
-	 *
-	 * @return bool True if Woo Multi-Currency is not enabled, false otherwise.
-	 */
-	public static function check_multi_currency_disabled() {
-		if ( class_exists( 'WOOMC\MultiCurrency\App' ) ) {
```

Comment:
At least with the latest version of the Multi-Currency extension, this check wasn't working anyway: the class exists at `WOOMC\App`.